### ISSUE TITLE
Develop story like a community rather than merging every PR

### ISSUE THREAD

**Issue by AseedUsmani** [KartSriv/Story#22]
Going by the comments on devrant, the story may go nowhere if we do not maintain quality.

Just like every Open Sourced project, let's have a team of reviewers go through a PR before it is merged.

**Comment by Hampusm** [comment]
I agree :+1:

**Comment by Hampusm** [comment]
Its currently and KartSriv reviewing pull requests but we might need more

**Comment by AseedUsmani** [comment]
I can volunteer, and I know someone who'd be good with this. Also, before merging a PR we can give the requester suggestions as well.

**Comment by Hampusm** [comment]
Yeah. It would also be pretty nice if there was issues with big ideas for the story that everyone can talk about

**Comment by KartSriv** [comment]
@AssedUsmani @Hampusm That's a nice suggestion Assed but I can't really work on it right now cause I am AFK this weekend. So, @Hampusm will guide you through all the stuff you need. Thanks - Karthik Srivijay

**Comment by Hampusm** [comment]
@KartSriv could you just make him a collaborator

**Comment by Hampusm** [comment]
📌 pinned it

**Comment by Hampusm** [comment]
@AseedUsmani Make sure too watch this repository and you can review pull requests. I Will make sure @KartSriv makes you have a collaborator when hes online again
